reference/marketplace-api.yaml

```diff
@@ -1216,31 +1251,34 @@ components:
           type: string
         customer_id:
           type: string
+          description: The platform assigned ID used to identify this contact record.
         phone:
+          type: array
           items:
             type: string
-          type: array
         state:
           type: string
         address:
           type: string
         email:
+          type: array
           items:
             type: string
-          type: array
         origins:
+          type: array
           items:
             type: string
-          type: array
         assigned_rep_first_name:
           type: string
         assigned_rep_last_name:
           type: string
         permission_to_contact:
           type: string
           format: date-time
+          description: The last time the contact granted permission to contact them.
```

Comment:
It's actually either when the user was created or updated and the 'permission to contact' checkbox was checked... or if they were created or updated via API, and the flag was set. Currently we provide no method for a contact to actually provide permission to contact them, that is up to the vendor to collect. 
So this statement is slightly missleading perhaps - since the permission currently is being set by the Vendor or via Business App by the Business... not guaranteed to be by the contact?

Reply:
So is there any value in returning a timestamp? Should this just always be a boolean?

Reply:
Probably fine to return the timestamp, in the case that there was an update to just that field that could potentially be useful, even potentially for compliance and privacy verification requests. I'm not worried about the format of info delivery, but rather am being nit-picky about the description stating that the 'contact granted permission to contact them' when that isn't necessarily true.

What I'm hinting at is probably a larger discussion(which is ongoing with legal, but stalled), but in terms of this pr is a matter of semantics where I just want to ensure we cover our buts.

This 'permission_to_contact' flag as a mechanism is not good enough for handling gdpr or various privacy concerns without updating the terms of service of the Vendors who integrate with us, while at the same time we are validating that any vendor that integrates with the Customer List is actually implementing a valid method for gathering customer contact confirmation. Neither of which we do at this time.

While we're getting all that stuff figured out(and with CH throwing uncertainty into the future of this anyhow), best to not pretend that this flag provides any type of guarantees or legal protection, or misrepresent it in any fashion.